### ISSUE TITLE
Does it work for pirated version??

### ISSUE THREAD

**Issue by colinlayen12** [topher-au/SnakeBite#8]
OK so i tried to install infinite haven mod but it says that I need to restor original game files under snakebite setings when I do that nothing happens help.

**Comment by YomiRev** [comment]
This is actually interesting as a concept because I would prefer to crack my copy and play on a lower patch than the current one. If I did so it would register as a pirated copy correct? Even though I actually own the game actually several copies of it across several platforms lol :) oh and Snakebite is awesome keep up the awesome work! Also I don’t think that this mod manager is intended for pirated copies of it although your question from 2017 made me wonder about the nature of patch compatibility with it.

**Comment by KeyC0de** [comment]
The start screen of SnakeBite:

![Clipboard01](https://user-images.githubusercontent.com/17159000/74974267-95b2b580-542d-11ea-8fc1-bfbb1ff67bba.jpg)

**Comment by colinlayen12** [comment]
It does but i think some mods do only skins but jesus 2017 haha im on
consoles more now but yeah the skin mods work perfectly

**Comment by Whit3C0br4** [comment]
Ok i know Snakebite it's not intended to work on non-official copies of the game but there isn't any chance of making it work?
I dont have money to spend on buying the game and i've been playing it "pirated" for a long time
And i'll really love to put mods on it, can anyone help me?

**Comment by 8wpt** [comment]
> Ok i know Snakebite it's not intended to work on non-official copies of the game but there isn't any chance of making it work? I dont have money to spend on buying the game and i've been playing it "pirated" for a long time And i'll really love to put mods on it, can anyone help me?

Yes, there is. Simulate folder structure of Steam and Snakebite will be fooled. x:\Steam\Steamapps\Common\MGS_TPP

**Comment by sweetrevengeAC** [comment]
> > Ok i know Snakebite it's not intended to work on non-official copies of the game but there isn't any chance of making it work? I dont have money to spend on buying the game and i've been playing it "pirated" for a long time And i'll really love to put mods on it, can anyone help me?
> 
> Yes, there is. Simulate folder structure of Steam and Snakebite will be fooled. x:\Steam\Steamapps\Common\MGS_TPP

can you help me? a quick step by step should do the work, you will save me if you can help me

**Comment by 8wpt** [comment]
you must have atleast 1 steam game for this to work, first go to your steam folder ( x:\Steam\Steamapps\Common) and create a folder named "MGS_TPP" now go to the folder where you have the cracked game and copy all the files it has in it and move them to the MGS_TPP folder you created, once that's finished continue with the snakebite installation

**Comment by 8wpt** [comment]
> you must have atleast 1 steam game for this to work, first go to your steam folder ( x:\Steam\Steamapps\Common) and create a folder named "MGS_TPP" now go to the folder where you have the cracked game and copy all the files it has in it and move them to the MGS_TPP folder you created, once that's finished continue with the snakebite installation

oops i dind't quote your reply

**Comment by gigahertz88** [comment]
> you must have atleast 1 steam game for this to work, first go to your steam folder ( x:\Steam\Steamapps\Common) and create a folder named "MGS_TPP" now go to the folder where you have the cracked game and copy all the files it has in it and move them to the MGS_TPP folder you created, once that's finished continue with the snakebite installation

Did that and it keeps showing me this
![obrázok_2022-06-02_180408631](https://user-images.githubusercontent.com/106763208/171674401-1de030dc-20d9-4259-adc1-8e5f6fd1775c.png)

**Comment by 8wpt** [comment]
> > you must have atleast 1 steam game for this to work, first go to your steam folder ( x:\Steam\Steamapps\Common) and create a folder named "MGS_TPP" now go to the folder where you have the cracked game and copy all the files it has in it and move them to the MGS_TPP folder you created, once that's finished continue with the snakebite installation
> 
> Did that and it keeps showing me this ![obrázok_2022-06-02_180408631](https://user-images.githubusercontent.com/106763208/171674401-1de030dc-20d9-4259-adc1-8e5f6fd1775c.png)

hm, i have no idea about that error
maybe try reinstalling the game

**Comment by gigahertz88** [comment]
> > > you must have atleast 1 steam game for this to work, first go to your steam folder ( x:\Steam\Steamapps\Common) and create a folder named "MGS_TPP" now go to the folder where you have the cracked game and copy all the files it has in it and move them to the MGS_TPP folder you created, once that's finished continue with the snakebite installation
> > 
> > 
> > Did that and it keeps showing me this ![obrázok_2022-06-02_180408631](https://user-images.githubusercontent.com/106763208/171674401-1de030dc-20d9-4259-adc1-8e5f6fd1775c.png)
> 
> hm, i have no idea about that error maybe try reinstalling the game

will i loose my progress?

**Comment by 8wpt** [comment]
> > > > you must have atleast 1 steam game for this to work, first go to your steam folder ( x:\Steam\Steamapps\Common) and create a folder named "MGS_TPP" now go to the folder where you have the cracked game and copy all the files it has in it and move them to the MGS_TPP folder you created, once that's finished continue with the snakebite installation
> > > 
> > > 
> > > Did that and it keeps showing me this ![obrázok_2022-06-02_180408631](https://user-images.githubusercontent.com/106763208/171674401-1de030dc-20d9-4259-adc1-8e5f6fd1775c.png)
> > 
> > 
> > hm, i have no idea about that error maybe try reinstalling the game
> 
> will i loose my progress?

as far as i'm aware, no (i already tested it myself)

**Comment by gigahertz88** [comment]
> 

Alright... So bassicly I uninstall the game and just run the setup again?

**Comment by 8wpt** [comment]
> > 
> 
> Alright... So bassicly I uninstall the game and just run the setup again?

yeah

**Comment by gigahertz88** [comment]
> 

ok, before I did this i tried it with GZ and it worked so im gonna do it with TPP now...

**Comment by Rartyu** [comment]
> > 
> 
> ok, before I did this i tried it with GZ and it worked so im gonna do it with TPP now...

you could be using an outdated version of snakebite or something near equal to that
Check application "MGSTPP" details for your game version and make sure your installed snakebite is for the intended version

**Comment by Rartyu** [comment]
admittedly, I don't know how the installer checks, so you could also try emulating your steam userdata on the game and everything else in the steam folder too.

**Comment by kevlarsteel** [comment]
guys I have Solution, just Rename your cracked game folder to MGS_TPP and it work for me

**Comment by 8wpt** [comment]
> guys I have Solution, just Rename your cracked game folder to MGS_TPP and it work for me

bro ur 2 years late

**Comment by mohamed-ahmed-soc** [comment]
> guys I have Solution, just Rename your cracked game folder to MGS_TPP and it work for me

you're big boss

**Comment by ruegaga** [comment]
> guys I have Solution, just Rename your cracked game folder to MGS_TPP and it work for me

Ow, man. You are briliant.

**Comment by OddBotlol** [comment]
hello guys i have a similar problem but it is a little diffirent.

Iam running a cracked version of mgsv tpp on windows 11 (my computer is unsupported i used a work around to get windows 11)
there was a problem with windows 11, fitgirl repacks crack of the game didnt work, it wouldnt launch so i had to use cpy's crack of the game.

im trying to use snake bite to play as quiet but when i use snake bite the game wont launch, i launch the game independently of snake bite and the game boots right up, im using the MQPM mod to play as quiet, her voice ques and running animation work but the skin hasnt changed, which iam preety sure is because the game wont launch with snake bite, i cant for the life of me figure out what the problem is, would really love to get some help.
